nodejs/week4/homework-template/src/frontend/pages/meals.js

```diff
@@ -0,0 +1,95 @@
+const createMeal = () => {
+  const title = document.querySelector('#title').value;
+  const description = document.querySelector('#description').value;
+  const location = document.querySelector('#location').value;
+  const reservation_time = document.querySelector('#reservation_time').value;
+  const max_reservation = document.querySelector('#max_reservation').value;
+  const price = document.querySelector('#price').value;
+
+  const created_date= Date.now()
+
+  if(!title || !description  || !location || !reservation_time  || !max_reservation || !price){
+    alert('Please enter information details')
+  } else {
+    postData(`/api/meals`, {
+      'title': title,
+      'description': description,
+      'location': location,
+      'reservation_time': reservation_time,
+      'max_reservation': max_reservation,
+      'price': price,
+      'created_date': created_date
```

Comment:
I think the idea with `created_date` is that the backend will take care of setting that.